libp2p/NodeTable.cpp

```diff
@@ -313,7 +313,8 @@ void NodeTable::evict(shared_ptr<NodeEntry> _leastSeen, shared_ptr<NodeEntry> _n
 	unsigned evicts = 0;
 	DEV_GUARDED(x_evictions)
 	{
-		m_evictions.push_back(EvictionTimeout(make_pair(_leastSeen->id,chrono::steady_clock::now()), _new->id));
+		EvictionTimeout evictTimeout{ _leastSeen->id,chrono::steady_clock::now(),_new->id };
```

Comment:
Please reformat to `EvictionTimeout evictTimeout{_leastSeen->id, chrono::steady_clock::now(), _new->id};`